conference/src/main/java/com/kyriba/conference/management/service/HallServiceImpl.java

```diff
@@ -1,47 +1,78 @@
 package com.kyriba.conference.management.service;
 
+import com.kyriba.conference.management.dao.HallRepository;
 import com.kyriba.conference.management.api.dto.HallRequest;
+import com.kyriba.conference.management.api.dto.HallResponse;
 import com.kyriba.conference.management.domain.Hall;
+import com.kyriba.conference.management.domain.exception.EntityNotFound;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.stream.StreamSupport.stream;
 
 
 @Service
-@Transactional
 public class HallServiceImpl implements HallService
 {
-  @Override
-  public Optional<Hall> find(Long id)
+  private final HallRepository hallRepository;
+
+
+  @Autowired
+  public HallServiceImpl(HallRepository hallRepository)
```

Comment:
Could be simplified with Lombok annotation

Reply:
done